consensus/dev/dev.go

```diff
@@ -85,15 +100,19 @@ func (d *Dev) run() {
 			return
 		}
 
-		// pick txn from the pool and seal them
+		// There are new transactions in the pool, try to seal them
 		header := d.blockchain.Header()
-		if err := d.do(header); err != nil {
+		if err := d.writeNewBlock(header); err != nil {
 			d.logger.Error("failed to mine block", "err", err)
 		}
 	}
 }
 
-func (d *Dev) do(parent *types.Header) error {
+// writeNewBLock generates a new block based on transactions from the pool,
+// and writes them to the blockchain
+func (d *Dev) writeNewBlock(parent *types.Header) error {
```

Comment:
Nice one, if anyone told me they had a `do` function in a `dev.go` I couldn't guess in 100 years what it might be doing. 🙂